crates/admin/src/cluster_controller/logs_controller.rs

```diff
@@ -8,24 +8,27 @@
 // the Business Source License, use of this software will be governed
 // by the Apache License, Version 2.0.
 
+use rand::prelude::IteratorRandom;
+use rand::{thread_rng, RngCore};
```

Comment:
If you switch to `rustftm` nightly, you can use this config:

```
> cat ~/.config/rustfmt/rustfmt.toml
imports_granularity = "Module"
group_imports = "StdExternalCrate"  # nightly only
```

and it will automatically keep things in mostly the right shape, most of the time. (You'll also have to tell RustRover to use it, checkbox in settings.)

Reply:
Thanks for the pointer. I will try this out. W/o tooling support, I fear that I'll constantly fail at it.